designs/developer/developer-catalog-4.7/index.md

```diff
@@ -0,0 +1,178 @@
+- In the flow below, the user has selected All Items, and, therefore, no subcategories apply when the user arrives on the Helm Charts page.
+
+#### The user selects Helm charts type
+
+![Helm charts catalog](img/5-user-selects-helm.png)
+
+- User selects Helm charts type and is taken to the Helm charts catalog
+
+#### Helm charts catalog
+![Helm charts catalog](img/6-helm-catalog.png)
+
+- The chart repositories filter is introduced in the Helm charts catalog, and is multi-select.
+- The breadcrumb stem takes the user back to the generic developer catalog.
+- If the user has selected a category on the Helm charts page, when the user navigates back to the generic developer catalog that category selection should remain until deselected.
+
+### Path 2/2: User selects Helm Charts tile
+
+![Helm charts catalog](img/7-add-helm.png)
+
+- When the user selects the Helm chart card from the catalog, the user will be taken directly to the Helm charts catalog.
+
+![Helm charts catalog](img/6-helm-catalog.png)
+
+## Enhancements that apply across all catalog screens
+### Category interaction and behavior
+- Categories and subcategories that are empty should be hidden. For example, the event sources mockups at the bottom of this document show what it looks like when there are no categories and/or no filters.
+- When the user has selected a category or subcategory that subcategory should remain when the user drills down into type. If the user navigates away from Event sources and clicks into Helm charts card on the dev console the same category should be applied, unless the category is unavailable, and then no category selection is made.
+- Both the categories and types should have hover states. For example:
+
+![Helm charts catalog](img/8-hover-state.jpg)
+
+- When the user selects a category or subcategory the categories below the selected category should not disappear (as they do today).
+
+### Search experience
+- Every search should display “Filter by keyword...” in input field when no data has been input
+- PF search interactions:
+  - Field input: The user’s input immediately starts filtering down the list as the user types. There is no need for them to press enter.
+  - Clear: A clear button is easily accessible once the user begins typing to clear the field if necessary.
+- If there are no search results utilize OpenShift design convention [ADD link] for empty states shown below:
+
+### Sorting across the console
+- All navigation items and catalog items should default to A-Z sorting.
+- Sort by dropdown also lists A-Z, Z-A.
+
+![Helm charts catalog](img/9-sort-by-dropdown.png)
+
+### Interactions and text for type-specific catalogs
+- There should be 5 type specific catalogs
+  - Builder Images
+  - Event Sources
+  - Helm Charts
+  - Operator Backed
+- If another type is added post 4.x a new type specific catalog would also be needed
+Example screen from event sources:
```

Comment:
For some reason this line ran together with the previous in the preview I saw, but looks fine here 🤷‍♂️

Just figured I'd mention since I was momentarily confused.